lib/node_modules/@stdlib/stats/base/dists/betaprime/stdev/README.md

```diff
@@ -147,6 +147,96 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/betaprime/stdev.h"
+```
+
+#### stdlib_base_dists_betaprime_stdev( alpha, beta )
+
+Returns the standard deviation of a betaprime distribution.
+
+```c
+double out = stdlib_base_dists_betaprime_stdev( 1.0, 3.0 );
+// returns ~0.866
+```
+
+The function accepts the following arguments:
+
+-   **alpha**: `[in] double` first shape parameter
```

Comment:
@aayush0325 Please see https://github.com/stdlib-js/stdlib/pull/3354. Descriptions should end with periods.